### ISSUE TITLE
si.processes() returns all processes as unknown

### ISSUE THREAD

**Issue by vincenthawke** [sebhildebrandt/systeminformation#216]
I don't know if this is a bug, but `si.processes()` returns all processes as unknown.

System information:
OS Name: Microsoft Windows 10 Pro
Version:	10.0.17763 Build 17763
Processor: Intel(R) Core(TM) i9-9900K
BaseBoard: MPG Z390 GAMING PRO CARBON AC (MS-7B17)

```const fs  = require("fs");
const si = require('systeminformation');

(async () => {
    let processes = await si.processes();
    fs.writeFileSync("processes.json", JSON.stringify(processes));
})();

Below is the processes.json upper part.
{
    "all": 241,
    "running": 0,
    "blocked": 0,
    "sleeping": 0,
    "unknown": 241,
    "list": [
        {
            "pid": 0,
            "parentPid": 0,
            "name": "System Idle Process",
            "pcpu": 92.28259902427254,
            "pcpuu": 0,
            "pcpus": 92.28259902427254,
            "pmem": 0.00002388689174568152,
            "priority": 0,
            "mem_vsz": 56,
            "mem_rss": 8,
            "nice": 0,
            "started": "2019-04-11 20:43:05",
            "state": "unknown",
            "tty": "",
            "user": "",
            "command": "System Idle Process"
        },
        {
            "pid": 4,
            "parentPid": 0,
            "name": "System",
            "pcpu": 0.6611224635987302,
            "pcpuu": 0,
            "pcpus": 0.6611224635987302,
            "pmem": 0.006986915835611843,
            "priority": 8,
            "mem_vsz": 200,
            "mem_rss": 2340,
            "nice": 0,
            "started": "2019-04-11 20:43:05",
            "state": "unknown",
            "tty": "",
            "user": "",
            "command": "System"
        },
...

**Comment by sebhildebrandt** [comment]
@vincenthawke ... investigating it ... will inform you tomorrow about my test results.

**Comment by sebhildebrandt** [comment]
@vincenthawke ... I made some tests and the underlying WMIC command `process get /value` returns an `ExecutionState` as well as `Status`. Both are documented do return some information about the current state of the process. Unfortunately both are always empty (NULL) on all of my test environments. So at the moment I do not see a possibility to retrieve reliable information about the process state. Therefore I decided to set this to `unknown` (if the value is empty).
